src/lando/main/models/repo.py

```diff
@@ -96,10 +139,12 @@ def get_mapping(cls) -> dict[str, "Repo"]:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.path = self.system_path or self.get_system_path()
+
         if self.is_hg_repo:
-            self.hg = HgRepo(self.system_path or self.get_system_path())
+            self.vcs = HgVcs(self.path)
         else:
-            self.hg = None
+            self.vcs = None
```

Comment:
I think this'll change to a getter instead: this will make the Repo object a pure(r) ORM interface, which can provide an concrete Scm object to the landing_worker. This will allow us to get rid of the proxy function on the Repo object altogether, and let the worker interact directly with the Scm interface.